[2024-03-18T07:57:05.434+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: two_separate_groups.bigquery_external_table_3_task manual__2024-03-18T06:40:31.356042+00:00 [queued]>
[2024-03-18T07:57:05.492+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: two_separate_groups.bigquery_external_table_3_task manual__2024-03-18T06:40:31.356042+00:00 [queued]>
[2024-03-18T07:57:05.499+0000] {taskinstance.py:2193} INFO - Starting attempt 3 of 3
[2024-03-18T07:57:05.571+0000] {taskinstance.py:2214} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_3_task> on 2024-03-18 06:40:31.356042+00:00
[2024-03-18T07:57:05.594+0000] {standard_task_runner.py:60} INFO - Started process 1776 to run task
[2024-03-18T07:57:05.610+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'two_separate_groups', 'bigquery_external_table_3_task', 'manual__2024-03-18T06:40:31.356042+00:00', '--job-id', '288', '--raw', '--subdir', 'DAGS_FOLDER/ingest_data_to_gcs.py', '--cfg-path', '/tmp/tmpwapos86j']
[2024-03-18T07:57:05.611+0000] {standard_task_runner.py:88} INFO - Job 288: Subtask bigquery_external_table_3_task
[2024-03-18T07:57:05.679+0000] {task_command.py:423} INFO - Running <TaskInstance: two_separate_groups.bigquery_external_table_3_task manual__2024-03-18T06:40:31.356042+00:00 [running]> on host 9958f290dc4a
[2024-03-18T07:57:05.783+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='two_separate_groups' AIRFLOW_CTX_TASK_ID='bigquery_external_table_3_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-18T06:40:31.356042+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-18T06:40:31.356042+00:00'
[2024-03-18T07:57:05.787+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-03-18T07:57:05.788+0000] {connection.py:269} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-03-18T07:57:05.790+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-03-18T07:57:06.383+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=two_separate_groups, task_id=bigquery_external_table_3_task, execution_date=20240318T064031, start_date=20240318T075705, end_date=20240318T075706
[2024-03-18T07:57:06.426+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-18T07:57:06.452+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
